Program.cs

```diff
@@ -0,0 +1,119 @@
+﻿Console.WriteLine("введите первое число");
+int m = int.Parse(Console.ReadLine());
+Console.WriteLine("введите второе число");
+int n = int.Parse(Console.ReadLine());
+int s = (n+m)%2;
+static bool sravnenie(ref int s,ref int m,ref int n)
+{
+    if (s == 0 & (m*n)%3 == 0)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+Console.WriteLine(sravnenie(ref s, ref m, ref n));
+
+Console.WriteLine("введите х");
+double x = double.Parse(Console.ReadLine());
+Console.WriteLine("введите y");
+double y = double.Parse(Console.ReadLine());
+static bool koordinati(ref double x, ref double y)
+{
+    if((x>= 2 & y>=0)|(x>=1 & y<=-1))
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+Console.WriteLine(koordinati(ref x, ref y));

```

Comment:
Те же советы по улучшению кода, что и выше:
```suggestion
static bool CheckCoordinates(double x, double y)
{
    return (x >= 2 && y >= 0) || (x >= 1 && y<= -1);
}

Console.WriteLine(CheckCoordinates(x, y));
```